src/data/pokemon/item_effects.h

```diff
@@ -325,42 +325,66 @@ const u8 gItemEffect_DireHit[8] = {
 };
 
 const u8 gItemEffect_XAttack[8] = {
+#ifdef BATTLE_ENGINE
+    [1] = ITEM1_X_ATTACK,
+#else
     [0] = 1,
+#endif
     [5] = ITEM5_FRIENDSHIP_LOW | ITEM5_FRIENDSHIP_MID,
     [6] = 1,
     [7] = 1,
 };
 
 const u8 gItemEffect_XDefend[8] = {
+#ifdef BATTLE_ENGINE
+    [1] = ITEM1_X_DEFEND,
```

Comment:
```suggestion
    [1] = ITEM1_X_DEFENSE,
```
This should also be changed in the BE PR where you defined a new value for this. So old one without IE can be called ITEM1_X_DEFEND and new one when IE is merged will be called ITEM1_X_DEFENSE.

Reply:
Like I mentioned above, renames to items, their constant labels and such should all be handled in a single PR for each branch, imo *(or well, for the BE and the IE specifically)*.

If that's not good, I'll apply the change though :eyes: